src/meta/src/hummock/compaction/picker/base_level_compaction_picker.rs

```diff
@@ -137,8 +137,8 @@ impl LevelCompactionPicker {
             min_compaction_bytes,
             // divide by 2 because we need to select files of base level and it need use the other
             // half quota.
-            std::cmp::min(
-                self.config.max_bytes_for_level_base,
+            std::cmp::max(
+                (self.config.max_bytes_for_level_base as f64 * 1.2) as u64,
```

Comment:
`max_compaction_bytes/2` will be larger than `self.config.max_bytes_for_level_base`. So this change makes no difference